cmd/format.go

```diff
@@ -1,9 +1,18 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/cobraext"
+	"github.com/elastic/elastic-package/internal/formatter"
+	"github.com/elastic/elastic-package/internal/packages"
+)
+
+// Flag "fail-fast" is used by commands: check, format
+const (
+	failFastFlagName        = "fail-fast"
+	failFastFlagDescription = "fail immediately if any file requires updates"
```

Comment:
Since these flags are used by multiple commands, should we move them to `internal/cobraext`, maybe in a `const.go` file or something?

Reply:
Good idea, fixed